lib/pengine/bundle.c

```diff
@@ -1587,10 +1587,7 @@ pe__bundle_replica_output_html(pcmk__output_t *out, GListPtr only_show,
                            replica->ipaddr);
     }
 
-    node = pe__current_node(replica->container);
-    if (pcmk__str_in_list(only_show, node->details->uname)) {
-        pe__common_output_html(out, rsc, buffer, node, options);
-    }
+    pe__common_output_html(out, rsc, buffer, node, options);
```

Comment:
don't need to pass only_show to these functions anymore

Reply:
Done.